driver/src/main/com/mongodb/reactivestreams/client/gridfs/GridFSBucket.java

```diff
@@ -556,10 +621,234 @@
      * @return a publisher with a single element, representing the amount of data written
      * @mongodb.server.release 3.6
      * @since 1.7
+     * @deprecated prefer {@link GridFSBucket#downloadToPublisher(ClientSession, String, GridFSDownloadOptions)} instead
      */
+    @Deprecated
     Publisher<Long> downloadToStream(ClientSession clientSession, String filename, AsyncOutputStream destination,
                                      GridFSDownloadOptions options);
 
+    /**
+     * Uploads the contents of the given {@code AsyncInputStream} to a GridFS bucket.
+     * <p>
+     * Reads the contents of the user file from the {@code source} and uploads it as chunks in the chunks collection. After all the
+     * chunks have been uploaded, it creates a files collection document for {@code filename} in the files collection.
+     * </p>
+     *
+     * @param filename the filename for the stream
+     * @param source   the Stream providing the file data
```

Comment:
Replace

   Stream => Publish

in all Javadoc

Reply:
Done - copy pasta